pkg/operator/ceph/object/user.go

```diff
@@ -79,6 +79,8 @@ func decodeUser(data string) (*ObjectUser, int, error) {
 	if len(user.Keys) > 0 {
 		rookUser.AccessKey = &user.Keys[0].AccessKey
 		rookUser.SecretKey = &user.Keys[0].SecretKey
+	} else {
+		return nil, RGWErrorBadData, errors.New("AccessKey and SecretKey are missing")
```

Comment:
Can you explain in which circumstances the store would be missing? It seems that we should through an error earlier.

Reply:
With my change unit test was crashed since it creation of admin user got failed for enabledashboard()

Reply:
> With my change unit test was crashed since it creation of admin user got failed for enabledashboard()

@leseb any suggestions?

Reply:
Not really, actually you are handling an error so it's good!